python-packages/middlewares/src/index.rst

```diff
@@ -0,0 +1,37 @@
+.. source for the sphinx-generated build/docs/web/index.html
+
+Python zero_ex.middlewares
+==========================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+.. automodule:: zero_ex.middlewares
+   :members:
+
+zero_ex.middlewares.local_message_signer
+----------------------------------------
+
+.. automodule:: zero_ex.middlewares.local_message_signer
+   :members:
+
+Example::
+
+    from web3 import Web3
```

Comment:
I actually think this example belongs in a docstring in `src/zero_ex/web3_middlewares/__init__.py`, and specifically, encoded as a doctest (prefix statements with `>>>`, and expected outputs with `...`; see an example of this in order_utils docstrings).  That way the example will be exercised during testing, when setup.py does `pytest.main(["--doctest-modules"])`.  And since we start up a ganache instance before that test run, you can use localhost:8545 for the RPC URL here.

Reply:
What is the best way to run tests locally? Currently running at module level and this module doe s not bring up ganache so it will fail if we try to use ganache.

Reply:
There is a custom command encoded in setup.py, so that you can run `./setup.py ganache` first and then `./setup.py test` after.  This is what our CI runs do.